release-notes/release-notes53.md

```diff
@@ -224,19 +224,24 @@ This section describes issues that users of {{site.productname}} 5.3 may encount
 
 **Outline**
 
-* [](#)
+* [Find and replace in selection feature not working as expected for inline editors and IE 11](#findandreplaceinselectionfeaturenotworkingasexpectedforinlineeditorsandie11)
 
 
-### 
+### Find and replace in selection feature not working as expected for inline editors and IE 11
```

Comment:
I'm not sure I'd say "not working as expected", as it is working, there's just no visual representation due to how browsers natively work. That being said, I can't think of a better term atm :shrug:

Reply:
No idea if it's better but maybe "Find and replace in selection feature missing content selection for inline editors and IE 11"?

Reply:
Yeah, that sounds better

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1537/commits/877c4459eeb8713c8e652aa83e444bfdcc7ae6f6